.github/workflows/static-analysis.datadog.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches: [master]
```

Comment:
Similar feedback here, is there a reason why you only want to try on the master branch right now?

Reply:
And also that you can use SCA here as well!